client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -35,17 +62,18 @@ const PaymentsActivityData: React.FC = () => {
 						) }
 					/>
 				}
+				amount={ totalPaymentsVolume }
 				reportLink={ getAdminUrl( {
```

Comment:
Sorry, this struck me later. We need to add a date filter to the link of the TPV as well as refunds as per design ( mentioned in Figma ).  I have a commit ready and can submit if you like. The changes look like this.  ( also sharing the change for refunds. 

```suggestion
				reportLink={ getAdminUrl( {
					page: 'wc-admin',
					path: '/payments/transactions',
					'date_between[0]': moment(
						getDateRange().date_start
					).format( 'YYYY-MM-DD' ),
					'date_between[1]': moment( getDateRange().date_end ).format(
						'YYYY-MM-DD'
					),
					filter: 'advanced',
				} ) }
```